### PR TITLE
updating app registration steps

### PR DESCRIPTION
**Pull request by madansr7**
updating app registration steps.

### CODE REVIEW

**Comment by Jackson-Woods** [comment]
@mark-szabo Can you please review this PR and accept+merge if appropriate? We're updating all MS Graph samples to reference the new app registration experience in the Azure portal.

File: README.md [hunk 1]
```diff
@@ -2,79 +2,98 @@
...
-* [Prerequisites](#prerequisites)
-* [Register the app](#register-the-app)
-* [Configure and run the sample](#configure-and-run-the-sample)
-* [Key components of the sample](#key-components-of-the-sample)
-* [Contributing](#contributing)
-* [Questions and comments](#questions-and-comments)
-* [Additional resources](#additional-resources)
-
-## Differences between ADAL and MSAL 
+- [Microsoft Graph Connect Sample for ASP.NET Core 2.1](#microsoft-graph-connect-sample-for-aspnet-core-21)
```
Comment (mark-szabo) [inline README.md]: Are you sure, you want to include all of these other things in the Table of Contents? I think only the old ones are relevant.

File: README.md [hunk 2]
```diff
@@ -2,79 +2,98 @@
...
-   >You'll use the application ID and secret to configure the app in Visual Studio.
+    1. After the screen has updated with the newly created client secret copy the **VALUE** of the client secret and store it temporarily as you will need it later.
 
-4. Configure Permissions for your application. **(Optional)**
+        > **Important:** This secret string is never shown again, so make sure you copy it now.
+        > In production apps you should always use certificates as your application secrets, but for this sample we will use a simple shared secret password.
+1. Click **Authentication** from the current blade navigation pane.
+    1. Select 'ID tokens'
 
-   >Note that we are not required to add permissions for reading user data and sending emails during the app registration as you would do with the [v1 endpoint (ADAL)](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-integrating-applications). The [Incremental and dynamic consent](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-v2-compare#incremental-and-dynamic-consent) capability of the v2 endpoint (MSAL) has made this step optional.
```
Comment (mark-szabo) [inline README.md:71]: Was this removed intentionally as not relevant anymore or just by a mistake? I think this is a very valuable information for v1-to-v2 switchers.

**Review by mark-szabo — CHANGES_REQUESTED** [review]
Sorry guys! I failed to notice the email notification... 😕

I'm absolutely supporting this PR. Left some questions in the review, can you please help conclude these questions quickly? 

Thanks a lot! 👍

File: README.md (hunk 2 above)
Comment (madansr7) [reply README.md:71]: yes, didn't intent to remove it. Checked with Philipe and will add the revised text.

File: README.md (hunk 1 above)
Comment (madansr7) [reply README.md]: @mark-szabo what about the other headers and content defined on the page? If it's on the page, shouldn't it be included in the ToC. Is your concern about the length of the ToC?

Comment (mark-szabo) [reply README.md]: Yea, I think if it is long, nobody will read it. I mean, a link to the repo itself in the ToC is not very usable info. Or the link to the ToC in the ToC...

IMHO only the relevant level 2 headers (##) should be included in the ToC.

File: README.md (hunk 2 above)
Comment (Jackson-Woods) [reply README.md:71]: @madansr7 did you get a chance to add the note for this PR?

File: README.md (hunk 1 above)
Comment (madansr7) [reply README.md]: reverted to previous ToC.

File: README.md (hunk 2 above)
Comment (mark-szabo) [reply README.md]: Thank you @madansr7!

File: README.md (hunk 1 above)
Comment (mark-szabo) [reply README.md]: Thank you @madansr7!

**Review by mark-szabo — APPROVED** [review]

**Pull request merged**